packages/appeals-service-api/__tests__/developer/external-dependencies/rest-apis/interactions/notify-interaction.ts

```diff
@@ -0,0 +1,104 @@
+import { Interaction } from './interaction';
+import { JsonPathExpression } from '../json-path-expression';
+const appConfiguration = require('../../../../../src/configuration/config');
+
+export class NotifyInteraction {
+	static getAppealSubmittedEmailForAppellantInteraction(
+		appeal: any,
+		appellantName: string,
+		lpaName: string
+	): Interaction {
+		return new Interaction('Send appeal successfully submitted email to appellant')
+			.setNumberOfKeysExpectedInJson(8)
+			.addJsonValueExpectation(
+				JsonPathExpression.create('$.template_id'),
+				appConfiguration.services.notify.templates[appeal.appealType]
+					.appealSubmissionConfirmationEmailToAppellant
+			)
+			.addJsonValueExpectation(JsonPathExpression.create('$.email_address'), appeal.email)
+			.addJsonValueExpectation(JsonPathExpression.create('$.reference'), appeal.id)
+			.addJsonValueExpectation(JsonPathExpression.create('$.personalisation.name'), appellantName)
+			.addJsonValueExpectation(
+				JsonPathExpression.create("$.personalisation['appeal site address']"),
+				appeal.appealSiteSection.siteAddress.addressLine1 +
+					'\n' +
+					appeal.appealSiteSection.siteAddress.addressLine2 +
+					'\n' +
+					appeal.appealSiteSection.siteAddress.town +
+					'\n' +
+					appeal.appealSiteSection.siteAddress.county +
+					'\n' +
+					appeal.appealSiteSection.siteAddress.postcode
+			)
+			.addJsonValueExpectation(
+				JsonPathExpression.create("$.personalisation['local planning department']"),
+				lpaName
+			)
+			.addJsonValueExpectation(
+				JsonPathExpression.create("$.personalisation['link to pdf']"),
+				`${process.env.APP_APPEALS_BASE_URL}/document/${appeal.id}/${appeal.appealSubmission.appealPDFStatement.uploadedFile.id}`
+			);
+	}
+
+	static getAppealSubmittedEmailForLpaInteraction(
+		appeal: any,
+		lpaName: string,
+		lpaEmail: string
+	): Interaction {
+		const templateVariables: any[] = [];
+		const dateRegex = new RegExp(
+			/\d{2} (January|February|March|April|May|June|July|August|SeptemberOctober|November|December) \d{4}/
+		);
+
+		if (appeal.appealType == '1001') {
+			templateVariables.push({ LPA: lpaName });
+			templateVariables.push({ date: dateRegex });
+		} else if (appeal.appealType == '1005') {
+			templateVariables.push({
+				'loca planning department': lpaName
```

Comment:
Looks like a typo here "loca"

Reply:
The typo is in horizon, this is mocking horizon

Reply:
Aye, this is quite funny @guiribeirokc 😂 There's a typo in the variable declared on Notify, and this was one of those "it was like that when we got here moments, so we left it as it is for now! We may want to create a ticket to rectify this though (if it hasn't already been created)?